### DISCUSSION TITLE
403 Forbidden

### DISCUSSION THREAD

**Discussion by Bold014** [tweepy/tweepy#2093] [Questions] [answered]
453 - You currently have Essential access which includes access to Twitter API v2 endpoints only. If you need access to this endpoint, you’ll need to apply for Elevated access via the Developer Portal. You can learn more here: https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#v2-access-leve

With elevated access gone, how do I fix this???

**Comment by Harmon758** [comment, marked as answer]
https://twittercommunity.com/t/announcing-new-access-tiers-for-the-twitter-api/188728
